automated/linux/armnn/armnn-unit-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
+usage() {
+    echo "Usage: $0 [-s <true|false>]" 1>&2
+    exit 1
+}
+
+while getopts "s:a:b:" o; do
+  case "$o" in
+    s) SKIP_INSTALL="${OPTARG}" ;;
+    a) LINK_SNAPSHOT_32BIT="${OPTARG}" ;;
+    b) LINK_SNAPSHOT_64BIT="${OPTARG}" ;;
+    *) usage ;;
+  esac
+done
+
+! check_root && error_msg "You need to be root to run this script."
+create_out_dir "${OUTPUT}"
+
+pkgs="ntp wget gcc g++ python3 python3-pip"
+dhclient
+install_deps "${pkgs}" "${SKIP_INSTALL}"
+
+#(GET WHICH PLATFORM)
+detect_abi
+
+if [ "$abi" == "armeabi" ]
```

Comment:
I think you can drop the if statement if you pass '-O my-armnn.tar.xz' to wget...

Then you can do 'tar xf my-armnn.tar.xz'
BASEDIR can't that be the same also?